inventory/aix_lvgs_inventory

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019 tribe29 GmbH - License: GNU General Public License v2
+# This file is part of Checkmk (https://checkmk.com). It is subject to the terms and
+# conditions defined in the file COPYING, which is part of this source code package.
+
+# Example input:
+#
+#[Volume_Groups_Active]
+#rootvg
+#usmvg
+#[Volume_Groups_Details]
+#cobackvg:
+#LV NAME             TYPE       LPs     PPs     PVs  LV STATE      MOUNT POINT
+#oraprodbacklv       jfs2       792     1584    4    closed/syncd  /oracle/prod/backup
+#loglv04             jfs2log    1       2       2    closed/syncd  N/A
+#free_alt_rootvg:
+#LV NAME             TYPE       LPs     PPs     PVs  LV STATE      MOUNT POINT
+#rootvg:
+#LV NAME             TYPE       LPs     PPs     PVs  LV STATE      MOUNT POINT
+#hd5                 boot       1       2       2    closed/syncd  N/A
+#hd6                 paging     240     480     2    open/syncd    N/A
+#livedump            jfs2       4       8       2    open/syncd    /var/adm/ras/livedump
+#lroot               jfs2       16      32      2    open/syncd    /root
+#sysdumplv00         sysdump    100     100     1    open/syncd    N/A
+#usmvg:
+#LV NAME             TYPE       LPs     PPs     PVs  LV STATE      MOUNT POINT
+#usrLocal            jfs2       64      128     2    closed/syncd    /usr/local
+#usrLclItslb         jfs2       8       16      2    closed/syncd    /usr/local/itslb
+#
+
+import re
+
+
+def parse_aix_lvgs(info):
+
+    header_active, header_details = u"[Volume_Groups_Active]", u"[Volume_Groups_Details]"
+    if info[0][0] != header_active:
+        raise Exception('Expecting a (possibly empty) list of active volume groups starting with ' +
```

Comment:
Please raise a specific exception type. ValueError seems an appropriate choice in this case.